README.md

```diff
@@ -34,9 +34,9 @@ Libocpp provides a complete implementation of OCPP 1.6. The implementation of OC
   - [Unit testing](#unit-testing)
   - [Building with FetchContent instead of EDM](#building-with-fetchcontent-instead-of-edm)
 
-## Feature Support
+## OCPP 1.6 Support
```

Comment:
Please check the headline level (currently ### Feature Profile Support OCPP 2.0.1 is sup-chapter of ## OCPP 1.6 Support) and adjust the table of contents accordingly

Reply:
Done.